dev/conductor/core/lib/src/state.dart

```diff
@@ -180,7 +180,15 @@ String phaseInstructions(pb.ConductorState state) {
     case ReleasePhase.PUBLISH_CHANNEL:
       return 'Issue `conductor next` to publish your release to the release branch.';
     case ReleasePhase.VERIFY_RELEASE:
-      return 'Release archive packages must be verified on cloud storage: ${luciConsoleLink(state.releaseChannel, 'packaging')}';
+      return <String>[
```

Comment:
I think you should either move this to the next phase `RELEASE_COMPLETED` (note we call this function at the end of the `conductor next` flow), or also add this text the prompt that we have the user acknowledge before proceeding.

Reply:
Otherwise lgtm

Reply:
Moved instructions and updated prompt